urbanairship-accengage-react-native/README.md

```diff
@@ -0,0 +1,43 @@
+# Airship Accengage React Native
+
+A React Native module for Airship's iOS and Android SDK.
+
+### Resources
+
+* [Getting started guide](http://docs.airship.com/platform/react-native/)
+* [API docs](http://docs.airship.com/reference/libraries/react-native/latest/index.html)
+
+### Issues
+
+Please visit https://support.airship.com/ for any issues integrating or using this module.
+
+### Requirements:
+ - Xcode 11+
+ - iOS: Deployment target 11.0+
+ - Android: minSdkVersion 16+, compileSdkVersion 28+
+ - React Native >= 0.60.0
+ - React Native cli >= 2.0.1
+
+## Install
+
+```
+# using yarn
+yarn add urbanairship-accengage-react-native
+
+# using npm
+npm install urbanairship-accengage-react-native --save
+```
```

Comment:
I'd break these into two blocks, since the user needs to use only one of the two.